### ISSUE TITLE
Undirected PageRank

### ISSUE THREAD

**Issue by szarnyasg** [ldbc/ldbc_graphalytics#166]
Is undirected PageRank included in the competition? It is a rather odd choice to run PR on an undirected graph. While it's certainly doable, it's not supported out-of-the-box by some graph analytical engines, so it might be required to use a directed graph with edges added to both directions.

https://github.com/ldbc/ldbc_graphalytics/blob/049b16494336cb76bc99a25a0e82c139cd67fe1a/graphalytics-validation/src/main/java/science/atlarge/graphalytics/validation/algorithms/pr/PageRankValidationTest.java#L86

It is mentioned in the specs as part of the test suite:

![image](https://user-images.githubusercontent.com/1402801/38768533-a151fde0-3ff5-11e8-830d-c0ef26327dcf.png)

**Comment by thegeman** [comment]
Both undirected PageRank and directed LCC are included in the benchmark. Although they may not be as widely used as their directed/undirected counterparts, there were several reasons to include them.

1. We wanted to have all algorithms defined for both directed and undirected graphs to avoid undue bias towards either type of graph. Such a bias could be in either the number of algorithms or the complexity of the algorithms (if we included PR for directed and LCC for undirected).
2. Even if a specific combination of algorithm and directivity is not as widely used, we picked the algorithms in the benchmark to represent a variety of computation/communication patterns, so we believe the results are still meaningful.

I hope this clarifies our choices.
